migration/migrator/migrations/system/20220501171554_add_saml_options.py

```diff
@@ -0,0 +1,28 @@
+"""Migration for the Submitty system."""
+from collections import OrderedDict
+import json
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+    authentication_file = config.config_path / 'authentication.json'
+    with authentication_file.open('r+') as auth_file:
+        auth_info = json.load(auth_file, object_pairs_hook=OrderedDict)
+        if not 'saml_options' in auth_info:
+            auth_info['saml_options'] = {}
```

Comment:
Maybe adding the keys with default value here?  As the PHP might throw an error at [`site/app/models/Config.php@L343`](https://github.com/Submitty/Submitty/pull/8030/files#diff-ecb12eba4b1d073362c00e86f6a3ee5d70bf982eada6d72ff244abec74792940R343)

Reply:
If you go and run CONFIGURE_SUBMITTY.py it would then fill those values in. Also if you wanted to just raw edit this config file then you could the put the values in. I don't think it's necessary to have them here and this is similar to how LDAP was done.